messages/0.1.5.txt

```diff
@@ -0,0 +1,16 @@
+Welcome to the TypeScript plugin for Sublime Text!
+
+Details about this package and its feature can be found on its GitHub
+repository at https://github.com/Microsoft/TypeScript-Sublime-Plugin.
+
+v0.1.5 updates:
+	- Code base refactored for easier future maintenance
```

Comment:
Agreed.  How does our code refactoring affect users of the plugin?  They aren't maintaining it.